src/app/item-popup/DesktopItemActions.m.scss

```diff
@@ -129,8 +128,12 @@
   display: flex;
   align-content: center;
   justify-content: center;
+  align-items: center;
   height: 36px;
   width: 36px;
+  &:nth-child(n + 2) {
+    margin-left: 8px;
```

Comment:
Maybe give them all margin-left and then remove the margin from `:last-child`? `nth-child` always confuses me and I'm not sure how this works with 1 or 2 characters.

Reply:
i gotta push back a little and say this is modern css & simple math.
n+2 just means every element starting with the second.

last millennium's last/first child override hack is more bytes, more complex,
has a non-obvious purpose until you've read like 4 lines minimum,
and the lines can get separated or have selector precedence issues

Reply:
That's fair, I guess I can never remember how the rule works.